app/lib/widgets/cards/productCard/shared/price.dart

```diff
@@ -0,0 +1,127 @@
+import 'package:flex_ui/utils/extensions.dart';
+import 'package:flutter/material.dart';
+import 'package:widgetbook/widgetbook.dart';
+import 'package:widgetbook_annotation/widgetbook_annotation.dart' as widgetbook;
+import 'package:intl/intl.dart';
+
+enum Variant {
+  standard,
+  savings, // Defaults to text decoration linethrough and color 'disabled'
+}
+
+class Price extends StatelessWidget {
+  const Price({
+    super.key,
+    required this.price,
+    this.formatterCallback,
+    this.variant = Variant.standard,
+    this.textStyle,
+  });
+  final double price;
+  final String Function(double)? formatterCallback;
+  final Variant? variant;
+  final TextStyle? textStyle;
+  @override
+  Widget build(BuildContext context) {
+    final defaultTheme = Theme.of(context);
+// @TODO before merge - potentially add theme fallback?
+
+    final formattedPrice = formatterCallback != null
+        ? formatterCallback!(price)
+        : price.toString();
+
+    final Map<Variant, TextStyle> variantTextStyle = {
+      Variant.standard: const TextStyle(),
+      Variant.savings: TextStyle(
+        color: defaultTheme.disabledColor,
+        decoration: TextDecoration.lineThrough,
+      ),
+    };
+
+    return Text(
```

Comment:
Open question, do we want to add `Semantics` here? It will be screen-read by Flutter's default widgets.. we may want to override if/when we display both the `listPrice` and `salePrice`, etc?